make/tools.mk

```diff
@@ -62,7 +63,7 @@ ifeq (x86_64, $(HOST_ARCH))
 endif
 
 .PHONY: tools
-tools: bin/tools/helm bin/tools/kubectl bin/tools/kind bin/tools/cosign bin/tools/ginkgo bin/tools/cmrel bin/tools/release-notes bin/tools/controller-gen k8s-codegen-tools bin/tools/goimports bin/tools/gotestsum bin/tools/ytt bin/tools/yq
+tools: bin/tools/helm bin/tools/kubectl bin/tools/kind bin/tools/cosign bin/tools/ginkgo bin/tools/cmrel bin/tools/release-notes bin/tools/controller-gen k8s-codegen-tools bin/tools/goimports bin/tools/gotestsum bin/tools/rclone bin/tools/ytt bin/tools/yq
```

Comment:
Question: is this target ever used, i.e somewhere in the codebase? I guess given that these are for different use cases, there will likely never be a need to install i.e both kind and rclone in the same flow

Reply:
I don't think it's used, but this might be used if say we were creating a container image containing all of these tools and we wanted to install them all at once!

This was a very handy target when developing; it's worth keeping IMO because I can see how it'll be useful later (and I do use it occasionally when I'm testing that everything installs as expected)

Reply:
Thanks, that makes sense to me 👍🏼